### DISCUSSION TITLE
Where are the runner mappings defined?

### DISCUSSION THREAD

**Discussion by TWiStErRob** [actions/virtual-environments#6920] [General questions]
I found the definitions of the images:
https://github.com/actions/runner-images/blob/main/images/linux/ubuntu2004.json

I'm looking for where `ubuntu-20.04` (which is used in `run-as:`) is mapped to this file. I understand there might be multiple steps involved, for example this image is built, versiones and pushed to some repo. I'm most curious about the top level definition.

Maybe a simpler question could be, where is `ubuntu-latest` defined to point at `ubuntu-22.04`?

**Comment by mikhailkoliada** [comment]
Hello! It is out of scope of the repo as it is defined by an inner github's backend

**Reply by TWiStErRob** [reply]
So it is not listed in any other public repo you know either?
